constants.go

```diff
@@ -668,6 +668,16 @@ const (
 	// during the setup of Connect My Computer. The prefix is followed by the name of the cluster
 	// user. See teleterm.connectmycomputer.RoleSetup.
 	ConnectMyComputerRoleNamePrefix = "connect-my-computer-"
+
+	// SystemOktaRequesterRoleName is a name of a system role that allows
```

Comment:
Did you group some other unrelated changes in this PR? Based on the title I'm not sure I'd expect to see anything about Okta here.

Reply:
I added these changes in based off of discussion in the dev-identity slack. I should have updated the PR description here, apologies for that. The idea was "Extend access request max duration and make it possible to use it in Okta."